openshift_metrics/merge.py

```diff
@@ -91,6 +102,22 @@ def main():
 
     report_month = datetime.strftime(report_start_date, "%Y-%m")
 
+    if args.use_nerc_rates:
+        nerc_rates = load_from_url()
```

Comment:
This tripped me up for a bit because I was wondering how load_from_url() doesn't take any params until I realized it comes from `nerc_rates`. Maybe importing nerc rates and calling `nerc_rates.load_from_url()` would be better like you are doing for `get_value_at()` below.

Reply:
@hakasapl Good point! I was also overwriting the name of the module which would have been confusing even if it didn't break things. I have fixed this now.